### PR TITLE
git fundamentals

### PR DESCRIPTION
**Pull request by lohithkumar2002**
# explain git commands
-  git add, git commit etc.

### CODE REVIEW

**Comment by Hemasri-P** [comment]
Is there  any other git commands

File: Bahubali-2.txt [hunk 1]
```diff
@@ -1,3 +1,6 @@
 Kattappa killed Bahubali 😭😭😭 for Raja matha 💀💀💀
 
-Due to Raja matha gone crazy 🎊🎊💀
\ No newline at end of file
+Due to Raja matha gone crazy 🎊🎊💀
+
+
+The story continues
```
Comment (ragavkumarv) [inline Bahubali-2.txt]: Please add `.` at the end

File: Bahubali-2.txt [hunk 2]
```diff
@@ -1,3 +1,6 @@
 Kattappa killed Bahubali 😭😭😭 for Raja matha 💀💀💀
 
-Due to Raja matha gone crazy 🎊🎊💀
\ No newline at end of file
+Due to Raja matha gone crazy 🎊🎊💀
```
Comment (ragavkumarv) [inline Bahubali-2.txt]: Add `!` at the end

Comment (Rishisairi) [inline Bahubali-2.txt:3]: Add reason for killing

**Review by ragavkumarv — CHANGES_REQUESTED** [review]
Please address all the comments mentioned

File: Bahubali-2.txt (hunk 1 above)
Comment (lohithkumar2002) [reply Bahubali-2.txt]: resolved

File: Bahubali-2.txt (hunk 2 above)
Comment (lohithkumar2002) [reply Bahubali-2.txt]: done

**Review by ragavkumarv — APPROVED** [review]
LGTM!!!

**Comment by nikhilrajganta** [comment]
Good work @lohithkumar2002 🥳

**Pull request merged**